arch/armv8m/jmp.S

```diff
@@ -0,0 +1,121 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libphoenix
+ *
+ *
+ * _setjmp, _longjmp, setjmp, sigsetjmp
+ *
+ * Copyright 2018, 2019 Phoenix Systems
+ * Author: Kamil Amanowicz, Aleksander Kaminski, Jakub Sarzyński
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+ /* setjmp env buffer description
+  * Offset  Description
+  * 0       Signal mask flag - if != 0 then next value stores a signal mask
+  * 4       Signal mask
+  * 8       SP register
+  * 12-43   r4-r11 registers
+  * 44      LR register
+  * 48-79   d8-d15 registers
+  * 80      fpscr register
+  */
+
+#define __ASSEMBLY__
+
+.thumb
+.syntax unified
+
+.text
+.align 2
```

Comment:
Alignment might cause some performance penalty. IMHO better to set it to 4, to make it predictable

Reply:
Do we want to leave 4 for other arm architectures?

Reply:
Other arm archs aren't in the scope of this PR. Imho it's best to change alignment to 4 everywhere eventually, as @lukileczo found out it's faster indeed

Reply:
Oh I see that it has been changed recently